coder_sniffer/DrupalPractice/Sniffs/Objects/StrictSchemaDisabledSniff.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * DrupalPractice_Sniffs_Objects_StrictSchemaDisabledSniff.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+
+namespace DrupalPractice\Sniffs\Objects;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\AbstractVariableSniff;
+
+/**
+ * Checks that $strictConfigSchema is not set to FALSE in test classes.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+class StrictSchemaDisabledSniff extends AbstractVariableSniff
+{
+
+    /**
+     * The name of the variable in the test base class to disable config schema checking.
+     */
+    const STRICT_CONFIG_SCHEMA_NAME = '$strictConfigSchema';
+
+
+    /**
+     * Processes this test, when one of its tokens is encountered.
+     *
+     * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
+     * @param int                         $stackPtr  The position of the current token
+     *                                               in the stack passed in $tokens.
+     *
+     * @return void
+     */
+    public function processMemberVar(File $phpcsFile, $stackPtr)
+    {
+        $tokens = $phpcsFile->getTokens();
+
+        if (($tokens[$stackPtr]['content'] === static::STRICT_CONFIG_SCHEMA_NAME) && ($this->isTestClass($phpcsFile, $stackPtr) === true)) {
+            $find = array(
+                     T_FALSE,
+                     T_TRUE,
+                     T_NULL,
+                     T_SEMICOLON,
+                    );
+            $next = $phpcsFile->findNext($find, ($stackPtr + 1));
```

Comment:
This can go wrong if a class declares variables like `protected $foo, $bar = 5;`. So I think you also need to search for ','. We should also add a test case for this.

Reply:
I added a test for it, but didn't need to update the code since it eventually ends with a semi-colon regardless.